doc/nrf/ug_thingy53_gs.rst

```diff
@@ -41,10 +41,16 @@ If you are using a mobile device, one of the following operating systems:
 Installing the required software
 ********************************
 
-To work with the Nordic Thingy:53 firmware using a computer, install `nRF Connect for Desktop`_.
+To work with the Nordic Thingy:53 firmware:
+
+* using a computer, install `nRF Connect for Desktop`_.
+  After installing and starting the application, install the Programmer app.
+* using a mobile device, install the `nRF Programmer`_ mobile application from the corresponding application store.
+
+When using a computer to work with with the Nordic Thingy:53 firmware, install `nRF Connect for Desktop`_.
 After installing and starting the application, install the Programmer app.
 
-To work with the Nordic Thingy:53 firmware using a mobile device, install the `nRF Programmer`_ mobile application from the corresponding application store.
+When using a mobile device to work with the Nordic Thingy:53, install the `nRF Programmer`_ mobile application from the corresponding application store.
```

Comment:
This section now has two alternatives for presenting the information.

Opinions on which looks better needed.

Reply:
@wiba-nordic I guess this conversation can be resolved?

Reply:
Yes, I'll take out the bullet point version.